docarray/array/storage/annlite/backend.py

```diff
@@ -33,7 +33,11 @@ class AnnliteConfig:
 class BackendMixin(BaseBackendMixin):
     """Provide necessary functions to enable this storage backend."""
 
-    TYPE_MAP = {'str': 'TEXT', 'float': 'float', 'int': 'integer'}
+    TYPE_MAP = {
+        'str': ('TEXT', str),
```

Comment:
Could we make it a name tuple ? Otherwise it is not straightforward to understand the rest of the code

Reply:
done